pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -852,6 +856,15 @@ func (gce *GCECloud) createTargetPool(name, serviceName, region string, hosts []
 	for _, host := range hosts {
 		instances = append(instances, makeHostURL(gce.projectID, host.Zone, host.Name))
 	}
+	if len(instances) > maxInstancesPerTargetPool {
```

Comment:
I meant healthy (kubernetes) nodes. We don't health check target pools behind a service of type=loadbalancer anyway, so GCE wont know they're not healthy.